mock-api/api/survey-unit/kqhlz2/GET.js

```diff
@@ -14,7 +13,8 @@ module.exports = function (request, response) {
 	try {
 		fs.accessSync(filePath);
 	} catch (err) {
-		return response.status(404);
+		response.statusCode = 302;
```

Comment:
You PR is about 301 code, you have `302` here.

Do you have a test in your mock-api for the `301` code.